users-service/service.yml

```diff
@@ -26,4 +26,7 @@ messages:
 
 
 dependencies:
-  - 'mongo'
+  mongo:
+    prod: 'ds143588.mlab.com:43588'
+    dev: 'mongo:27017'
+    test: 'localhost:27017'
```

Comment:
Yes, they do. The way it works for testing is a little tricky. 

Seeing as we want mongo running inside a docker container, while the test code itself is still running locally, we have to publish the `27017` port of the mongo container that is created so that it can be accessed through `localhost`. 

`exo-test` handles this, creating a `test-mongo` container with that port published if the service has `mongo` in its `dependencies` field of `service.yml`. So for testing on ci servers, we will want to either user `exo-test` to run the tests, or keep doing them the same way we have, but add `mongodb`  in the `.travis.yml` `services` section.